### PR TITLE
feature: add portfolio rebalancing strategy

### PR DESCRIPTION
**Pull request by narumiruna**

### CODE REVIEW

File: pkg/strategy/rebalance/strategy.go [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+	value := p.TotalValue()
+	for currency, asset := range p.Assets {
+		weights[currency] = asset.MarketValue().Div(value)
+	}
+	return weights
+}
+
+func (p *Portfolio) PrintValue() {
+	value := p.TotalValue()
+	log.Infof("portfolio value: %f %s", value.Float64(), p.BaseCurrency)
```
Comment (c9s) [inline pkg/strategy/rebalance/strategy.go:57]: you can use market.BaseCurrencyFormatter and market.QuoteCurrencyFormatter

File: pkg/strategy/rebalance/strategy.go [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+		log.Infof("sum of weights: %f != 1.0", sum.Float64())
+	}
+
+	for currency, w := range s.TargetPortfolio {
+		s.TargetPortfolio[currency] = w.Div(sum)
+	}
+
+}
+
+func (s *Strategy) Rebalance(ctx context.Context, orderExecutor bbgo.OrderExecutor, session *bbgo.ExchangeSession) {
```
Comment (c9s) [inline pkg/strategy/rebalance/strategy.go:128]: if verbose than shows "Rebalancing..." here?

**Review by c9s — APPROVED** [review]

File: pkg/strategy/rebalance/strategy.go (hunk 2 above)
Comment (narumiruna) [reply pkg/strategy/rebalance/strategy.go:128]: 沒有特別想放什麼，拿掉好了XD

**Review by c9s — APPROVED** [review]
awesome, thanks!

**Pull request merged**